### PR TITLE
Fix cad.onshape.com

### PR DESCRIPTION
**Pull request by TilCreator**
Onshape needs some more complex inverting, else the sketch lines are black like the background.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -3416,6 +3416,16 @@ INVERT
 
 ================================
 
+cad.onshape.com
+
+IGNORE INLINE STYLE
+#canvas
+
+CSS
+canvas { filter: invert(1) hue-rotate(180deg) !important; background: #fff !important; }
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:3425]:
```suggestion
INVERT
canvas

IGNORE INLINE STYLE
#canvas
```

**Review by Gusted — CHANGES_REQUESTED** [review]

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (TilCreator) [reply src/config/dynamic-theme-fixes.config]: The suggested change didn't work as the background color needs to be changed. But I used the recommended changes to simplify it.

**Comment by TilCreator** [comment]
This page can be used for testing: https://cad.onshape.com/documents/354728f5f71464187c094ae2/w/f727c899b6bbf374df58de10/e/e39054473ac3eed9d31449d4?renderMode=0&uiState=63e52244bb71826649fc0970

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -3416,6 +3416,16 @@ INVERT
 
 ================================
 
+cad.onshape.com
+
+INVERT
+canvas
+
+CSS
+canvas { background: #fff !important; }
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:3425]:
```suggestion
canvas {
    background-color: #fff !important;
}
```

**Comment by TilCreator** [comment]
@Gusted Is there still something for me todo?

**Comment by TilCreator** [comment]
This seams to be broken now. I will reopen a new PR, as these changes here are quite outdated now. sry

**Pull request closed without merging**